packages/realm/src/app-services/EmailPasswordAuth.ts

```diff
@@ -86,15 +90,23 @@ export class EmailPasswordAuth {
 
   /**
    * Sends an email to the user for resetting the password.
-   * @param credential The email details to send the reset to
-   * @param credential.email - The email address of the user to re-send a confirmation for.
+   * @param details The email details to send the reset to
```

Comment:
```suggestion
   * @param details - The email details to send the reset to
```
I like the `-` after the argument name in the `@param` directive for readability.  Most likely missing in other places.  Maybe a lint rule?

Reply:
https://github.com/gajus/eslint-plugin-jsdoc/blob/main/.README/rules/require-hyphen-before-param-description.md

Reply:
I suggest leaving that for a future PR.